### PR TITLE
[meta] Merge history from beta 11 to alpha 12

### PR DESCRIPTION
**Pull request by mcdurdin**

### CODE REVIEW

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/138020:id/kvk/index.html)

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/138024:id/kvk/index.html)

File: web/history.md [hunk 1]
```diff
@@ -2,13 +2,27 @@
 
 ## 12.0 alpha
 
-## 11.0 alpha
-* Add `setNumericLayer()` for embedded platforms to change OSK to numeric layer.
-* Fixes issue where file extensions are upper-case, e.g. ".TTF"
-* Fixes keyboard layout issues after mobile device rotations. (#248) (#970)
-* Fixes issue with oversized key text on some keyboards. (#382)
-* Fixes issue with diacritics not displaying on some keyboards.  (#1070)
-* Adds support for Promises to init() and setActiveKeyboard(). (#100)
```
Comment (glasseyes) [inline web/history.md]: There is an extra entry here now so there is a conflict

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/138082:id/kvk/index.html)

**Pull request merged**